test/distributed/org/apache/cassandra/distributed/shared/ClusterUtils.java

```diff
@@ -659,9 +696,12 @@ public static void waitForCMSToQuiesce(ICluster<IInvokableInstance> cluster, Epo
                 if (skip)
                     continue;
 
-                if (cluster.get(j).isShutdown())
+                IInvokableInstance inst = cluster.get(j);
+                if (inst.isShutdown())
                     continue;
-                Epoch version = getClusterMetadataVersion(cluster.get(j));
+                Epoch version = getClusterMetadataVersion(inst);
+                if (fetchLogWhenBehind && version.getEpoch() < awaitedEpoch.getEpoch())
+                    version = fetchLogFromCMS(inst, awaitedEpoch);
```

Comment:
this was feedback from sam/marucs but was not actually the root cause... just good to have

When we see nodes are out of sync we *hope* that something will trigger the node to catch up... there is another method in this class that triggers messages to get sent which will trigger that... instead I just ask to fetch the logs directly... 

by default this behavior is off, so only `TopologyMixupTestBase` triggers this code path

Reply:
+1 as long as this is off by default